hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/master/MasterStatusTmpl.jamon

```diff
@@ -224,6 +225,11 @@ AssignmentManager assignmentManager = master.getAssignmentManager();
           <& AssignmentManagerStatusTmpl; assignmentManager=master.getAssignmentManager()&>
         </%if>
         <%if !master.isInMaintenanceMode() %>
+          <%java>
+          while (master.getMasterCoprocessorHost() == null) {
+            Threads.sleep(50);
```

Comment:
Let's also print some message which we can removed after getting out of this `while` loop?

Reply:
I researched it and it seems I can’t

Reply:
Would the whole page not load (display) when master.getMasterCoprocessorHost() == null ?

Reply:
Good question, the only concern is what will the page display when `master.getMasterCoprocessorHost()` is null?
Is this situation reproable by any chance?

Reply:
As of commit 82d09904f217096b564e3dba2261aee6c2274516 in master branch, I don't see the master.getMasterCoprocessorHost().findCoprocessor() call below.

One option is to find the commit which removed master.getMasterCoprocessorHost().findCoprocessor() call.

Another option is to use code similar to existing one:
```
                           <td><% master.getMasterCoprocessorHost() == null ? "[]" :
                              java.util.Arrays.toString(master.getMasterCoprocessors()) %></td>
```

Reply:
my negligence, there are some similar problems and solutions in front